charts/tidb-backup/templates/scripts/_start_restore.sh.tpl

```diff
@@ -21,3 +21,19 @@ downloader \
   {{ .Values.restore.options }}
 {{- end }}
 
+{{- if .Values.restore.ceph }}
+downloader \
+  --cloud=ceph \
+  --bucket={{ .Values.restore.ceph.bucket }} \
+  --endpoint={{ .Values.restore.ceph.endpoint }} \
+  --srcDir={{ .Values.restore.ceph.srcDir }} \
+  --destDir=${dataDir}
+
+/loader \
+  -d ${dataDir}/{{ .Values.restore.ceph.srcDir }} \
```

Comment:
This section looks the same as above except for srcDir. I think we can move `srcDir` up a level so that it will be `.Values.restore.srcDir`

Reply:
Good idea. I've updated it.